multiplatform/fetch_fastdds_manual/action.yml

```
@@ -0,0 +1,45 @@
+name: fetch_fastdds_manual
+
+inputs:
+
+  foonathan_memory_vendor_branch:
+    description: Branch, tag or commit of eProsima/foonathan_memory_vendor repository. Check available branches in https://github.com/eProsima/foonathan_memory_vendor.
+    required: false
+    default: master
+
+  fastcdr_branch:
+    description: Branch, tag or commit of eProsima/Fast-CDR repository. Check available branches in https://github.com/eProsima/Fast-CDR.
+    required: false
+    default: master
+
+  fastdds_branch:
+    description: Branch, tag or commit of eProsima/Fast-DDS repository. Check available branches in https://github.com/eProsima/Fast-DDS.
+    required: false
+    default: master
+
+  destination_workspace:
+    description: 'Workspace where the repositories are downloaded'
+    required: false
+    default: '${{ github.workspace }}'
+
+runs:
+  using: composite
+  steps:
+
+    - name: Run in ubuntu
+      uses: eProsima/eProsima-CI/ubuntu/fetch_fastdds_manual@feature/manual_fastdds_build
```

Before merging:

```suggestion
      uses: eProsima/eProsima-CI/ubuntu/fetch_fastdds_manual@main
```

Apply also to other occurrences in this PR.